Modix.Services/Moderation/ModerationService.cs

```diff
@@ -914,9 +914,9 @@ private async Task<IEnumerable<GuildRoleBrief>> GetRankRolesAsync(ulong guildId)
                 }))
                 .Select(r => r.Role);
 
-        private async Task RequireSubjectRankLowerThanModeratorRankAsync(ulong guildId, ulong subjectId)
+        private async Task RequireSubjectRankLowerThanModeratorRankAsync(ulong guildId, ulong moderatorId, ulong subjectId)
         {
-            if (!await DoesModeratorOutrankUserAsync(guildId, AuthorizationService.CurrentUserId.Value, subjectId))
+            if (!await DoesModeratorOutrankUserAsync(guildId, moderatorId, subjectId))
```

Comment:
515 and 867 *call* 919.

917-921 can't just use the `AuthorizationService` value because it's the responsibility of the thing that calls this method to know who the user is, not this method's. I think we'll run into an exception at runtime complaining about not having a user ID if we use `AuthorizationService` here, since the handler runs in its own scope that doesn't have that data available.